finality-aleph/src/lib.rs

```diff
@@ -258,3 +259,8 @@ pub struct AlephConfig<B: Block, H: ExHashT, C, SC> {
     pub external_addresses: Vec<String>,
     pub validator_port: u16,
 }
+
+pub trait GetBlockchainBackend<B: Block> {
```

Comment:
Rather than having a trait that only returns another trait, it's likely better to implement some analogue of the original trait on the object you are implementing this one on by calling the returning method internally.

This comment is probably borderline incomprehensible, but we already talked about this, so I'm mostly leaving it here to mark the discussion publicly. <_<"